README.md

```diff
@@ -82,7 +82,25 @@ Creates a new user in your OPC installation. The user's private key will be retu
 
 Deletes the given OPC user.
 
-### knife opc user edit USERNAME
+### knife opc user edit USERNAME (options)
+
+- `-i FILENAME`, `--input FILENAME`: Reads the user information (`username`, `email`, `display_name`, `first_name`, `last_name`, `middle_name`) from the input FILENAME and updates the given OPC user. This file is expected to be in the json format.
```

Comment:
```suggestion
- `-i FILENAME`, `--input FILENAME`: Reads the user information (`username`, `email`, `display_name`, `first_name`, `last_name`, `middle_name`) from the input FILENAME and updates the given OPC user. Knife will accept a JSON file.
```

Reply:
@IanMadd I updated the changes.Please have a look.